src/components/App.tsx

```diff
@@ -103,13 +103,6 @@ export default class App extends React.Component<AppProps, AppState> {
     };
   }
 
-  changeViewType = (): void => {
```

Comment:
I don't think it's appropriate to make changes to things other than **configuring TSLint**.

Reply:
Sorry to hear that, I'll delete it on the fly